app/build.gradle

```diff
@@ -48,6 +48,7 @@ dependencies {
     debugCompile 'com.squareup.leakcanary:leakcanary-android:1.5'
     releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5'
     testCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5'
+    compile 'com.android.support.constraint:constraint-layout:1.0.0-beta4'
```

Comment:
Oooooooh I knew this day would come!!! 😄

To get the Travis build working, add [these lines of code](https://github.com/SUPERCILEX/Robot-Scouter/blob/master/.travis.yml#L13-L16).

PS: the latest version is `1.0.2` I believe.